impl/src/test/kotlin/nl/nuts/consent/bridge/listener/CordaStateChangeListenerIntegrationTest.kt

```diff
@@ -109,6 +119,19 @@ class CordaStateChangeListenerIntegrationTest {
                 }
             }.start()
 
+            // nats server
+            val config = EmbeddedNatsConfig.Builder()
+                .withNatsServerConfig(
+                    NatsServerConfig.Builder()
+                        .withServerType(ServerType.NATS_STREAMING)
+                        .withPort(4222)
```

Comment:
to make this test a bit more stable, you could use ServerSocket to find an open port:

```
new ServerSocket(0).getPort()
```
(don't forget to close it)